<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Channel Select</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>dMTmin</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>dt[ns]</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Selection</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>nTAC</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3" colspan="2">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Micro-time resoltion of the histogram</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Effective number of TAC-channels in the histogram.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Operators&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Logical&lt;/span&gt;: &amp;amp;, |, ~.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Comparison&lt;/span&gt;: &amp;lt;, &amp;lt;=, ==, !=, &amp;gt;=, &amp;gt;.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Unary arithmetic&lt;/span&gt;: -.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Binary arithmetic&lt;/span&gt;: +, -, *, /, **, %.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Example&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Selecting channel 0 and channel 8&lt;/p&gt;&lt;p&gt;(ROUT == 0) | (ROUT == 8)&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;See&lt;/span&gt;&lt;/p&gt;&lt;p&gt;http://pytables.github.io/usersguide/condition_syntax.html&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>(ROUT==0)&amp;(TAC&lt;3000)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>TAC div</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Micro-time divisor. The micro-times are divided by this number to coarsen the time-scale.</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>64</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>128</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>256</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>nPh</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Used channels in the histogram.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="2">
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Number of photons in the histogram</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Ch</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>If this option is on only photons with a macro-time bigger than the specified macro-time are considered in the histogram.</string>
        </property>
        <property name="text">
         <string>invert</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="checkBox">
        <property name="toolTip">
         <string>If this is on the minimum macro-time filter is used in the construction of the histogram.</string>
        </property>
        <property name="text">
         <string>on/off</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Minimum distance between photons in number of macro-time steps. All photons with a distance bigger than the minimum distance are not considered in the construction of the histogram.</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>500.000000000000000</double>
        </property>
        <property name="value">
         <double>200000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>make decay</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
